### ISSUE TITLE
Add to Calendar pulls empty (0 byte) ical.ics file

### ISSUE THREAD

**Issue by PRIMETSS** [aspnet/live.asp.net#126]
### Title
Add to Calendar pulls empty (0 byte) ical.ics file

### Functional impact
When selecting 'Add to Calendar' on home page, a zero byte ical.ics is downloaded.
Tested with multiple browsers.

### Minimal repro steps
Live site has this issue.

### Expected result
Get a file with .ics content.

### Actual result
File is zero bytes, empty.

### Further technical details
So I pulled down the repo, and not wanting to setup secrets and Azure stuff, couldnt get to Admin pages to create the ShowDetails.json file.
Modified HomeController to force a create of the ShowDetails.json file

 `var lsd = new LiveShowDetails() { AdminMessage = "Message", LiveShowEmbedUrl = "http://localhost", NextShowDateUtc = DateTime.Now, LiveShowHtml = "http://localhost" };
 var r =  _liveShowDetails.SaveAsync(lsd);`

Removed the AdminMessage, LiveShowHtml and LiveShowEmbedUrl as that prevents the Add To Calendar HTML from rendering.

ShowDetails.json (edited)

> {"NextShowDateUtc":"2017-03-27T09:22:54.3354571+10:30"}

Ran site locally and I DID get a ical.ics file with contents....
So basically looks like just an issue with files on Production site?

oh and Works as expected at
https://ms-asp-standup-staging.azurewebsites.net/ 

So got to be something pretty basic??

**Comment by hishamco** [comment]
IMHO if it's working on staging environment, perhaps @DamianEdwards needs to push the latest changes to production environment to make it works as expected

**Comment by bitbonk** [comment]
It also applies to the webcal option. Only the google calendar option works.
